compiler/test/compilable/vcg-ast-arraylength.d.cg

```diff
@@ -0,0 +1,406 @@
+module vcg_ast_arraylength;

```

Comment:
a`.cg` file should be in `extra-files` of the appropriate `test` directory, see e.g. `extra-files/checkaction-context-assert-0.d.cg`